### PR TITLE
[EPMRPP-38535] Add error log condition finding items by auto analyze status

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/analyzer/strategy/AutoAnalyzedCollector.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public AutoAnalyzedCollector(TestItemRepository testItemRepository, LogRepositor
 
 	@Override
 	public List<Long> collectItems(Long projectId, Long launchId, String login) {
-		List<Long> itemIds = testItemRepository.selectIdsByAutoAnalyzedStatus(true, launchId);
+		List<Long> itemIds = testItemRepository.selectIdsByAutoAnalyzedStatusWithErrorLogs(true, launchId);
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/analyzer/strategy/AutoAnalyzedCollector.java]: see commons-dao

Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/core/analyzer/strategy/AutoAnalyzedCollector.java]: done

File: src/main/java/com/epam/ta/reportportal/core/analyzer/impl/AnalyzerServiceImpl.java [hunk 2]
```diff
@@ -184,16 +184,18 @@ private void analyzeLaunch(Launch launch, List<TestItem> toAnalyze, IndexLaunch
...
-				RelevantItemInfo relevantItemInfo = updateTestItemIssue(projectId, analyzed, testItem);
-				TestItemActivityResource after = TO_ACTIVITY_RESOURCE.apply(testItem, projectId);
-
-				testItemRepository.save(testItem);
-				messageBus.publishActivity(new ItemIssueTypeDefinedEvent(before, after, analyzerInstance, relevantItemInfo));
-				ofNullable(after.getTickets()).ifPresent(it -> messageBus.publishActivity(new LinkTicketEvent(before,
-						after,
-						analyzerInstance
-				)));
+				if (!testItem.getItemResults().getIssue().getIssueType().getLocator().equals(analyzed.getLocator())) {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/analyzer/impl/AnalyzerServiceImpl.java]: could it be null somewhere here ? for example test item without issue?

Comment (ihar-kahadouski) [reply src/main/java/com/epam/ta/reportportal/core/analyzer/impl/AnalyzerServiceImpl.java]: I think no. Items without issue couldn't be analyzed. Only items with issue collects before analysis

**Pull request merged**